composer.json

```diff
@@ -13,14 +13,13 @@
     "homepage": "https://github.com/ipinfo/ipinfolaravel",
     "keywords": ["Laravel", "ipinfolaravel"],
     "require": {
-        "illuminate/support": "^7.0|^8.0|^9.0|^10.0|^11.0",
+        "illuminate/support": "^7.0|^8.0|^9.0|^10.0|^11.0|^12.0",
         "ipinfo/ipinfo": "^3.2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.6",
+        "phpunit/phpunit": "^12.0",
         "mockery/mockery": "^1.4.2",
-        "orchestra/testbench": "^8.36",
-        "sempro/phpunit-pretty-print": "^1.3.0",
```

Comment:
`sempro/phpunit-pretty-print` was used nowhere so I removed it, it's also not maintained anymore. It doesn't even support Laravel 10.